README.md

```diff
@@ -27,11 +27,8 @@ chaining,
 [intelligent planning](docs/PLANNERS.md)
 capabilities out of the box.
 
-![image](https://user-images.githubusercontent.com/371009/221739773-cf43522f-c1e4-42f2-b73d-5ba84e21febb.png)
-
 Semantic Kernel supports and encapsulates several design patterns from the latest
-in AI research, such that developers can infuse their applications with complex
-[skills](docs/SKILLS.md) like [prompt](docs/PROMPT_TEMPLATE_LANGUAGE.md)
+in AI research, such that developers can infuse their applications with  [plug-ins](https://learn.microsoft.com/semantic-kernel/howto/) like [prompt](docs/PROMPT_TEMPLATE_LANGUAGE.md)
```

Comment:
Shouldn't "plug-ins" be one word as "plugins"? https://openai.com/blog/chatgpt-plugins